messaging/messaging.go

```diff
@@ -274,6 +275,7 @@ type AndroidNotification struct {
 	TitleLocKey  string   `json:"title_loc_key,omitempty"`
 	TitleLocArgs []string `json:"title_loc_args,omitempty"`
 	ChannelID    string   `json:"channel_id,omitempty"`
+	Image        string   `json:"image,omitempty"`
```

Comment:
Lets rename to ImageURL to be consistent with the nomenclature used in Java SDK.